src/main/java/io/jenkins/plugins/coverage/targets/CoverageResult.java

```diff
@@ -435,22 +448,20 @@ public void setOwner(AbstractBuild<?, ?> owner) {
      * @return Value for property 'previousResult'.
      */
     public CoverageResult getPreviousResult() {
-//        if (parent == null) {
-//            if (owner == null) {
-//                return null;
-//            }
-//            Run<?, ?> prevBuild = BuildUtils.getPreviousNotFailedCompletedBuild(owner);
-//            CoberturaBuildAction action = null;
-//            while ((prevBuild != null) && (null == (action = prevBuild.getAction(CoberturaBuildAction.class)))) {
-//                prevBuild = BuildUtils.getPreviousNotFailedCompletedBuild(prevBuild);
-//            }
-//            return action == null ? null : action.getResult();
-//        } else {
-//            CoverageResult prevParent = parent.getPreviousResult();
-//            return prevParent == null ? null : prevParent.getChild(name);
-//        }
-        //TODO replace the CoberturaBuildAction to CoverageAction
-        return null;
+        if (parent == null) {
+            if (owner == null) {
+                return null;
+            }
+            Run<?, ?> prevBuild = BuildUtils.getPreviousNotFailedCompletedBuild(owner);
+            CoverageAction action = null;
+            while ((prevBuild != null) && (null == (action = prevBuild.getAction(CoverageAction.class)))) {
+                prevBuild = BuildUtils.getPreviousNotFailedCompletedBuild(prevBuild);
+            }
+            return action == null ? null : action.getResult();
+        } else {
+            CoverageResult prevParent = parent.getPreviousResult();
+            return prevParent == null ? null : prevParent.getChild(name);
+        }
```

Comment:
The commented out method, and this new added method look exactly the same (except for `CoverageAction` replacing `CoberturaBuildAction`).

Reply:
I don't know exactly what you mean. I think the commented out code has already been replaced by new added code in this pull request. Did I understand it wrong?

Reply:
Minor NIT, but if you want you can remove that commented part above. There's no need to have it.

Reply:
I think the commented part above is already been removed in there....